common/src/types/tasks/descriptors/update_wallet.rs

```diff
@@ -39,6 +40,10 @@ pub enum WalletUpdateType {
     PlaceOrder {
         /// The order to place
         order: Order,
+        /// The ID of the order
+        id: OrderIdentifier,
+        /// The matching pool to assign the order to
+        matching_pool: String,
```

Comment:
nit: maybe define `MatchingPoolId` or similar type alias. It fits in better with the precedent in our codebase and makes clearer interfaces that may otherwise return `String` down the line